### ISSUE TITLE
Some documentation still treats kernels as if they have an input_dim parameter

### ISSUE THREAD

**Issue by jjramsey** [GPflow/GPflow#1417]
In the notebook ["Multi-output Gaussian processes in GPflow"](https://gpflow.readthedocs.io/en/develop/notebooks/advanced/multioutput.html), there are Python statements like the following, where the initializers of kernels each have an integer argument:

```python
kernel = gpf.kernels.SharedIndependent(gpf.kernels.RBF(1) + gpf.kernels.Linear(1), output_dim=P)
kern_list = [gpf.kernels.RBF(D) + gpf.kernels.Linear(1) for _ in range(P)]
kern_list = [gpf.kernels.RBF(D) + gpf.kernels.Linear(D) for _ in range(L)]
```

Judging from the definition of `D`, it looks like those integer arguments are meant to be the `input_dim` arguments for the kernel initializers, but those arguments [no longer exist](https://gpflow.readthedocs.io/en/develop/notebooks/gpflow2_upgrade_guide.html#Kernel-Input-Dims).

I don't know if this problem occurs in other notebooks, but it wouldn't surprise me.

**Comment by st--** [comment]
Oufffff. Good catch! It was a huge effort porting all the notebooks across, and I wish we hadn't missed this one - but thanks for spotting it now! Would you like to help us out & contribute by fixing the notebook & opening a pull request ? Happy to help you out with the details if you like. There's also gpflow.slack.com if you'd want to join the chat! (Invite link in the README.md)

**Comment by jjramsey** [comment]
I can't promise that I'll have the time to do that, unfortunately.

**Comment by st--** [comment]
> I can't promise that I'll have the time to do that, unfortunately.

That's okay - if you don't manage to do it hopefully someone else will get to it at some point!

**Comment by st--** [comment]
This will be fixed by #1430. There was one other occurrence.
